src/pages/FormPage.js

```diff
@@ -1,11 +1,59 @@
-import React from 'react';
-import styled from 'styled-components';
 import Heading from '../components/Heading';
+import {useState} from 'react';
+import styled from 'styled-components';
+import JournalEntry from '../components/JournalEntry';
+import {writeToLocalStorage, loadFromLocalStorage} from '../util/localstorage';
+import dayjs from 'dayjs';
+
+const StyledTextArea = styled.textarea`
+  width: 100%;
+  height: 200px;
+`;
+
+const StyledButton = styled.button`
+  color: #797431;
+  border-radius: 15px;
+  padding: 5px;
+  margin: 10px;
+  text-align: center;
+`;
+
+const StyledLabel = styled.label`
+  display: none;
+`;
 
 export default function FormPage() {
+  const [textArea, settextArea] = useState('');
+  const onSubmit = () => {
+    const loadFromStorage = loadFromLocalStorage('JournalEntry') ?? [];
+    console.log(loadFromStorage);
+
+    writeToLocalStorage('JournalEntry', [...loadFromStorage, {text: textArea, date: dayjs()}]);
+  };
+
   return (
     <>
       <Heading text={'What are you grateful for today?'} />
+      <form>
+        <StyledLabel htmlFor="GratitudeJournal">Form for GratitudeJournal </StyledLabel>
```

Comment:
I am slightly confused: Is this the label of the text area, so that the user gets to know what he/she has to write into the field below, or is it a heading of the form?
I was wondering because the placeholder of the text area seems to have the content of what you want the user to write down.